openshift/release/common.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+# Collection of shared functions used to generate and prepare releases.
+
+ROOT_DIR=$(dirname $0)/../..
+
+readonly FAAS_VERSION=${FAAS_VERSION:-"main"}
+readonly FAAS_REPO=${FAAS_REPO:-"github.com/boson-project/faas"}
+
+# The vendor/ dir is omitted to be added separatelly
+UPDATED_FILES=$(cat <<EOT | tr '\n' ' '
+pkg/kn/root/plugin_register.go
+go.mod
+go.sum
+EOT
+)
+
+# Generates plugin_register.go file to enable plugin inlining
+generate_file() {
+  echo ":: Generating plugin_register.go file ::"
+  local faas_repo=$1
+  
+  cat <<EOF > "${ROOT_DIR}/pkg/kn/root/plugin_register.go"
```

Comment:
how about just carrying the file as and referencing it via its (predefined) path? this way any formatting issues etc can be fixed while the file is being updated in the editor than finding it in CI while the build may fail.

Reply:
At first I wanted to have everything contained as "all-in-one" script, but I like the idea to have the file visible outside of script exec context. 
Just to clarify a bit. Did you meant carry e.g. `./openshift/release/register_plugin.go` and cp to its working location here?